active_discussion/unions.md

```diff
@@ -1,3 +1,29 @@
 # Unions
 
-TBD
+## Outstanding questions
+
+* Is `#[repr(Rust)]` the bag-o-bytes union repr, or do we want to propose a new repr?
+    * *Discussion:* [#73: Validity of unions][#73]
+* The following questions are all implicitly answered if `#[repr(Rust)]` is the bag-o-bytes repr, but remain open if not:
+    * Do `#[repr(Rust)]` unions guarantee all fields at offset 0?
```

Comment:
Even bag-of-bytes unions could have fields at non-0 offsets. Not sure if there is any reason to do that, but this makes it sound like there is a necessary logical implication from bag-of-bytes to 0-offset, which I don't think is the case.